Stub for time.gmtime() looks incorrect

The following code:
```python3
from time import gmtime
(a, b, c, d, e, f, g, h, i) = gmtime()
print(a, b, c, d, e, f, g, h, i)
```
produces the following output:
```
$ python3 Test.py 
2019 9 30 16 29 49 0 273 0
$ mypy Test.py
Test.py:3: error: Too many values to unpack (9 expected, 11 provided)

$ mypy --version
mypy 0.730
$ python3 --version
Python 3.6.8
```
`reveal_type(gmtime)` produces the following output:
`Revealed type is 'def (secs: Union[builtins.float, None] =) -> Tuple[builtins.int, builtins.int, builtins.int, builtins.int, builtins.int, builtins.int, builtins.int, builtins.int, builtins.int, builtins.str, builtins.int, fallback=time.struct_time]`

--- a/stdlib/2and3/time.pyi
+++ b/stdlib/2and3/time.pyi
@@ -31,19 +31,24 @@ if sys.version_info >= (3, 3) and sys.platform != 'win32':
 if sys.version_info >= (3, 8) and sys.platform == "darwin":
     CLOCK_UPTIME_RAW: int
 
+class _struct_time(NamedTuple):
+    tm_year: int
+    tm_mon: int
+    tm_mday: int
+    tm_hour: int
+    tm_min: int
+    tm_sec: int
+    tm_wday: int
+    tm_yday: int
+    tm_isdst: int
+
+    @property
+    def n_fields(self) -> int: ...
+    @property
+    def n_sequence_fields(self) -> int: ...
+    @property
+    def n_unnamed_fields(self) -> int: ...
 if sys.version_info >= (3, 3):
-    class _struct_time(NamedTuple):
-        tm_year: int
-        tm_mon: int
-        tm_mday: int
-        tm_hour: int
-        tm_min: int
-        tm_sec: int
-        tm_wday: int
-        tm_yday: int
-        tm_isdst: int
-        tm_zone: str
-        tm_gmtoff: int
     class struct_time(_struct_time):
         def __init__(
             self,
@@ -63,17 +68,11 @@ if sys.version_info >= (3, 3):
             ],
             _arg: Any = ...,
         ) -> struct_time: ...
+        @property
+        def tm_zone(self) -> str: ...
+        @property
+        def tm_gmtoff(self) -> int: ...
 else:
-    class _struct_time(NamedTuple):
-        tm_year: int
-        tm_mon: int
-        tm_mday: int
-        tm_hour: int
-        tm_min: int
-        tm_sec: int
-        tm_wday: int
-        tm_yday: int
-        tm_isdst: int
     class struct_time(_struct_time):
         def __init__(self, o: _TimeTuple, _arg: Any = ...) -> None: ...
         def __new__(cls, o: _TimeTuple, _arg: Any = ...) -> struct_time: ...

